commands/install.cmd

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env bash
 [[ ! ${WARDEN_COMMAND} ]] && >&2 echo -e "\033[31mThis script is not intended to be run directly!" && exit 1
 
+if ! ( hash docker-compose 2>/dev/null); then
+    echo -e "\033[31mdocker-compose is not installed" && exit 1
+fi
+
+DOCKER_COMPOSE_VERSION="$(docker-compose -v | grep -o '[0-9.]*' | head -n1)"
```

Comment:
```suggestion
DOCKER_COMPOSE_VERSION="$(docker-compose -v | grep -oE '[0-9\.]+' | head -n1)"
```
Need the `-E` flag for this grep to work as expected on macOS. Tested this version of the command on Fedora 31 and macOS 10.15.